assets/scripts/palette/Palette.jsx

```diff
@@ -1,22 +1,17 @@
 import React, { useRef, useEffect } from 'react'
-import PropTypes from 'prop-types'
 import { IntlProvider } from 'react-intl'
 import { useSelector } from 'react-redux'
 import Scrollable from '../ui/Scrollable'
+import Tooltip, { useSingleton } from '../ui/Tooltip'
 import SegmentForPalette from '../segments/SegmentForPalette'
 import { getAllSegmentInfoArray } from '../segments/info'
 import { generateRandSeed } from '../util/random'
 import './Palette.scss'
 
-Palette.propTypes = {
-  handlePointerOver: PropTypes.func.isRequired,
-  handlePointerOut: PropTypes.func.isRequired,
-  handleScroll: PropTypes.func.isRequired
-}
-
 function Palette (props) {
```

Comment:
I'm gathering that the 'Palette' is the group of street segments available for the user to put on a street, right? A docstring would be great for people new to the codebase (up until I really dug into the review I assumed this component was for choosing colors or something)

Reply:
I'll update with the note, but I'm also open to suggestions for better naming conventions.